src/intl.ts

```diff
@@ -49,6 +49,13 @@ export const commonMessages = defineMessages({
   generalInformations: {
     defaultMessage: "General Information"
   },
+  imgageUploadError_title: {
```

Comment:
These names should also be camelCase to make it consistent with the rest naming convention. I don't see a reason why they aren't.

Reply:
Fixed.